Runtime/Core/Scripts/Animation/VoiceHandler.cs

```diff
@@ -165,10 +165,17 @@ public void PlayAudioClip(AudioClip audioClip)
 
         private float GetAmplitude()
         {
-            if (CanGetAmplitude)
+            if (CanGetAmplitude && AudioSource.clip.loadState == AudioDataLoadState.Loaded)
             {
-                var amplitude = 0f;
+                int currentPosition = AudioSource.timeSamples;
```

Comment:
Minor suggestion for consistency we can use ` var` here 
```suggestion
                var currentPosition = AudioSource.timeSamples;
```